src/Osma.Mobile.App/App.xaml.cs

```diff
@@ -62,7 +65,9 @@ private async Task Initialize()
 
         protected override void OnStart()
         {
-            // Handle when your app starts
+            AppCenter.Start("ios=" + AppConstant.IosAnalyticsKey + ";" +
```

Comment:
Should we surround this with a check so it doesn't happen in debug?
```
#if !DEBUG
     AppCenter.Start(
          "ios=" + AppConstant.IosAnalyticsKey + ";" +
          "android=" + AppConstant.AndroidAnalyticsKey + ";",
           typeof(Analytics), typeof(Crashes)
     );
#endif
```

Reply:
Yeap good idea, added